deploy/greptimedb/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: greptimedb-cluster
```

Comment:
seems missing roles and roleBinding?

Reply:
seems role/rolebinding is unnecessary since we dont have any CRD to operate